src/encoder.rs

```diff
@@ -1854,8 +1854,7 @@ fn encode_partition_topdown<T: Pixel>(
     PARTITION_SPLIT |
     PARTITION_HORZ |
     PARTITION_VERT => {
-      let num_modes = if partition == PARTITION_SPLIT { 1 }
-      else { 1 };
+      let num_modes = 1;
 
       if rdo_output.part_modes.len() >= num_modes {
```

Comment:
If we further merge this constant into the only statement where it is used, we can simply to the following:
```Rust
if !rdo_output.part_modes.is_empty() {
```

Reply:
Initially I thought that leaving this variable may clarify what actually has to be done, but you're right - no need to leave references to incomplete features. A person that will be able to complete the implementation won't be stopped by the absence of this variable.